DigitalOffice/Services/ProjectService/Validators/WorkersProjectIdsValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+using LT.DigitalOffice.ProjectService.Models;
+
+namespace LT.DigitalOffice.ProjectService.Validators
+{
+    public class WorkersProjectIdsValidator : AbstractValidator<WorkersIdsInProjectRequest>
+    {
+        public WorkersProjectIdsValidator()
+        {
+            RuleFor(workers => workers.ProjectId)
+                .NotEmpty()
+                .WithMessage("Project id");
+
+            RuleForEach(workers => workers.WorkersIds)
+                .NotEmpty()
+                .WithMessage("Each worker id");
```

Comment:
I thought, we shouldn't throw a message when use NotEmpty

Reply:
This first part message